bin/build/test/i18n/create_artifacts/test_common.clj

```diff
@@ -1,6 +1,6 @@
 (ns i18n.create-artifacts.test-common)
 
-(def singular-message-frontend
+(def ^:private singular-message-frontend
```

Comment:
For the missing docstring linting warnings, I made the function or var private if it's only used in its file.

Otherwise, I added a docstring.

Reply:
Let me know if you'd like a different approach 😄